client/tests/redefined_test.py

```diff
@@ -203,14 +187,14 @@ def average_user_transaction(transactions):
                 {
                     "name": "fraudulent",
                     "variant": "quickstart",
-                    "column": "IsFraud",
+                    "column": "IsFraud2",
```

Comment:
Is it safe to assume that this transactions file contains a `IsFraud2` column?

Reply:
it doesn't matter, we're never running it because it should fail already for resource redefined